mkt/webapps/cron.py

```diff
@@ -118,12 +119,15 @@ def clean_old_signed(seconds=60 * 60):
     """Clean out apps signed for reviewers."""
     log.info('Removing old apps signed for reviewers')
     root = settings.SIGNED_APPS_REVIEWER_PATH
-    for path in os.listdir(root):
-        full = os.path.join(root, path)
-        age = time.time() - os.stat(full)[stat.ST_ATIME]
-        if age > seconds:
-            log.debug('Removing signed app: %s, %dsecs old.' % (full, age))
-            shutil.rmtree(full)
+    for nextroot, dirs, files in walk_storage(root):
+        for fn in files:
+            full = os.path.join(nextroot, fn)
+            now = datetime.utcnow() if storage_is_remote() else datetime.now()
```

Comment:
Alternatively, before the loop:
`now_func = datetime.utcnow if storage_is_remote() else datetime.now`
then in the loop, just:
`now = nowfunc()`